### PR TITLE
Unifying static pages styles

### PR DESCRIPTION
**Pull request by andresgnlez**
Sorry for the amount of files involved in the PR, as you will understand, unify them means modify all of them :/

First commit appears because it's a branch created from `feature/responsive-faq-definitions`, ignore it please: second commit overrides most of this changes.

This PR is based on #218

### CODE REVIEW

**Comment by tiagojsag** [comment]
- [ ] Articles and publications => "publications" title spacing broken

**Comment by andresgnlez** [comment]
@tiagojsag 

About Articles page => Fixed on https://github.com/Vizzuality/GlobalFishingWatch/pull/247/commits/6e1a603cae5bd643a3dd75f93af9e259ab9f6d35

About the FAQ => https://app.zeplin.io/project.html#pid=578ca800634e4930188f3fd1&sid=57b2fedd6e0610e306013cb0 this is the new design (basically play with full-width lines and normal ones) Dani gave me after spoke with him.

Anyway, this PR is mainly to discuss with the rest of the team the way to organize this and get some feed back, I am not making responsive itself here. (That's why you weren't assigned 😆 )

**Comment by clementprdhomme** [comment]
This looks really good Andrés, I'm looking forward the responsive bits of the other PRs.

@nerik, @hectoruch, wanna add something?

File: app/styles/layout/l-static-page.scss [hunk 1]
```diff
@@ -1,45 +1,115 @@
...
+
+      @media #{$mq-tablet} {
+        padding: 80px 0 0;
+      }
+    }
   }
 
-  .testimonial > .video-quote {
-    margin: 30px 0 35px;
+  &.-tutorials {}
```
Comment (nerik) [inline app/styles/layout/l-static-page.scss]: remove empty rules

File: app/styles/layout/l-static-page.scss [hunk 2]
```diff
@@ -1,45 +1,112 @@
...
+        margin-bottom: 0;
+      }
+    }
+
+    :global .section-subtitle {
+      margin: 20px 0 5px;
+    }
+
+    .testimonial {
+      & + .testimonial {
```
Comment (nerik) [inline app/styles/layout/l-static-page.scss]: Not sure here, but `.testimonial:not(:first-child)` seems more logical for me

**Comment by nerik** [comment]
Apart for those 2 comments looks good 👍

File: app/styles/layout/l-static-page.scss (hunk 1 above)
Comment (andresgnlez) [inline app/styles/layout/l-static-page.scss]: Used in incoming PR's, that's why I keep them.

File: app/styles/layout/l-static-page.scss (hunk 2 above)
Comment (andresgnlez) [inline app/styles/layout/l-static-page.scss]: http://caniuse.com/#search=%3Anot not sure bro :/

Comment (andresgnlez) [inline app/styles/layout/l-static-page.scss]: Using `:first-child` you have to be sure this selector is scoped in the container who has all children, which is not the case. This way, it doesn't depend on the container. Not sure if I explained myself properly...

**Pull request merged**